mods/cnc/maps/nod05/nod05.lua

```diff
@@ -295,9 +295,9 @@ checkProduction = function(player)
 			end
 		end
 		if #UnitsType > 0 then
-			if (type == 'jeep' or type == 'mtnk') and not Factory.IsDead then
+			if (type == 'jeep' or type == 'mtnk') and not Factory.IsDead and Factory.Owner == gdi then
 				Factory.Build(UnitsType)
-			elseif (type == 'e1' or type == 'e2') and not Barracks.IsDead then
+			elseif (type == 'e1' or type == 'e2') and not Barracks.IsDead and Barracks.Owner == gdi then
```

Comment:
Above we check if we have units to produce. The units we want to be produced are stored in the `UnitsType` table. If we don't want to produce any, it's "length" is "lower than 1" (zero). Here we check for the actor types inside of the table, as infantry and vehicles aren't produced at the same facility. So, I guess this code is correct. ^^